dhis-2/dhis-services/dhis-service-administration/pom.xml

```diff
@@ -27,11 +27,38 @@
     </dependency>
     <dependency>
       <groupId>org.hisp.dhis</groupId>
-      <artifactId>dhis-service-validation</artifactId>
+      <artifactId>dhis-service-acl</artifactId>
     </dependency>
     <dependency>
       <groupId>org.hisp.dhis</groupId>
       <artifactId>dhis-service-program-rule</artifactId>
+      <scope>test</scope>
```

Comment:
can you please move the test scope dependencies into the test block `    <!-- Test -->` Since it seems to me that is a convention